n4js-libs/packages/org.eclipse.n4js.mangelhaft.reporter.ide/messages.md

```diff
@@ -1,4 +1,4 @@
-////
+<!--
```

Comment:
I don't think `<!-- -->` is the right comment syntax for Markdown. 
https://stackoverflow.com/questions/4823468/comments-in-markdown